kudos-webapps/src/main/webapp/vue-app/kudos/components/ActivityKudosReactionItem.vue

```diff
@@ -0,0 +1,187 @@
+          <span
+            v-if="fullname">
+            <span
+              :class="fullnameStyle"
+              class="text-truncate subtitle-2 my-auto">{{ fullname }}</span>
+            <span v-if="isExternal" class="muted">{{ externalTag }} </span>
+            <v-icon size="15" class="pl-2">fa-angle-right</v-icon>
+            <a :id="receiverId">
+              <span
+                v-if="kudosReceiverFullName"
+                :class="receiverNameStyle"
+                class="text-truncate subtitle-2 my-auto pl-1">{{ kudosReceiverFullName }} </span>
+            </a>
+          </span>
+          <relative-date-format
+            class="text-sm-caption text-sub-title"
+            :value="kudosElapsedTime" />
+        </div>
+      </a>
+      <template v-if="$slots.actions">
+        <slot name="actions"></slot>
+      </template>
+    </div>
+    <v-divider />
+  </div>
+</template>
+<script>
+const randomMax = 10000;
+
+export default {
+  props: {
+    kudos: {
+      type: Object,
+      default: null
+    },
+    retrieveExtraInformation: {
+      type: Boolean,
+      default: true,
+    },
+    boldTitle: {
+      type: Boolean,
+      default: () => false,
+    },
+    linkStyle: {
+      type: Boolean,
+      default: () => false,
+    },
+    avatarClass: {
+      type: String,
+      default: () => '',
+    },
+    tiptip: {
+      type: Boolean,
+      default: () => true,
+    },
+    tiptipPosition: {
+      type: String,
+      default: () => null,
+    },
+    title: {
+      type: String,
+      default: function() {
+        return `${this.title}`;
+      }
+    },
+    size: {
+      type: Number,
+      // eslint-disable-next-line no-magic-numbers
+      default: () => 37,
+    },
+  },
+  data () {
+    return {
+      userInformations: null,
+      senderId: `userAvatar${parseInt(Math.random() * randomMax)
+        .toString()
+        .toString()}`,
+      receiverId: `userAvatar${parseInt(Math.random() * randomMax)
+        .toString()
+        .toString()}`,
+      isExternal: false,
+
+    };
+  },
+  computed: {
+    inCommonConnections() {
+      return this.userInformations && this.userInformations.connectionsInCommonCount || 0;
+    },
+    sameUser() {
+      return this.userInformations && this.userInformations.username === eXo.env.portal.userName;
+    },
+    notConnected() {
+      return this.userInformations && !this.userInformations.relationshipStatus && !this.sameUser;
+    },
+    externalTag() {
+      return `( ${this.$t('userAvatar.external.label')} )`;
+    },
+    fullnameStyle() {
+      return `${this.boldTitle && 'font-weight-bold ' || ''}${!this.linkStyle && 'text-color' || ''}`;
+    },
+    receiverNameStyle() {
+      return `${this.boldTitle && 'font-weight-bold ' || ''}${this.linkStyle && 'text-color' || ''}`;
+    },
+    username() {
+      return this.kudos && this.kudos.senderId;
+    },
+    fullname() {
```

Comment:
To be defined if it's the sender's or receiver's full name?
```suggestion
    fullName() {
```